### PR TITLE
Test /wallet/create

### PR DESCRIPTION
**Pull request by Vavilen**
For [#639](https://github.com/skycoin/skycoin/issues/639)

### CODE REVIEW

**Comment by Vavilen** [comment]
@gz-c , [here](https://github.com/skycoin/skycoin/blob/6634757f5dbf49be580e8ad7a0862d25bcd2dedd/src/gui/wallet.go#L192) is an empty value for **wltName** param.

**Comment by gz-c** [comment]
This needs to be rebased on develop, I don't know why it isn't saying that the change to `gui.Spend` is not a conflict, that method has been removed

File: src/gui/wallet.go [hunk 1]
```diff
@@ -101,7 +109,7 @@ func walletBalanceHandler(gateway *daemon.Gateway) http.HandlerFunc {
 //  id: wallet id
 //	dst: recipient address
 // 	coins: the number of droplet you will send
-func walletSpendHandler(gateway *daemon.Gateway) http.HandlerFunc {
+func WalletSpendHandler(gateway Gatewayer) http.HandlerFunc {
```
Comment (gz-c) [inline src/gui/wallet.go]: do not capitalize these methods

File: src/gui/wallet_test.go [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+	"encoding/json"
+
+	"github.com/google/go-querystring/query"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/stretchr/testify/require"
+
+	"bytes"
+
+	"github.com/pkg/errors"
```
Comment (gz-c) [inline src/gui/wallet_test.go]: wrong errors package

File: src/gui/wallet_test.go [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"encoding/json"
+
+	"github.com/google/go-querystring/query"
```
Comment (gz-c) [inline src/gui/wallet_test.go]: do not use this

File: src/gui/wallet_test.go [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+	"github.com/stretchr/testify/mock"
+
+	"github.com/stretchr/testify/require"
+
+	"bytes"
```
Comment (gz-c) [inline src/gui/wallet_test.go]: sort the imports properly

**Comment by Vavilen** [comment]
@gz-c , fixed

File: src/gui/wallet_test.go [hunk 5]
```diff
@@ -699,3 +710,242 @@ func TestWalletBalanceHandler(t *testing.T) {
...
+		wltname                   string
+		scnN                      uint64
+		options                   wallet.Options
+		gatewayCreateWalletResult wallet.Wallet
+		gatewayCreateWalletErr    error
+		scanWalletAddressesResult wallet.Wallet
+		scanWalletAddressesError  error
+		responseBody              wallet.ReadableWallet
+	}{
+		{
```
Comment (gz-c) [inline src/gui/wallet_test.go]: `MethodGet`?

**Comment by gz-c** [comment]
merge conflict

**Pull request merged**